src/_grid.scss

```diff
@@ -150,6 +150,15 @@
 
 
 /**
+ * Modifier: all items match height of tallest item in a row
+ */
+
+.l-grid--equal-height > .l-grid__item {
+  display: flex;
```

Comment:
Cool :sunglasses:. 

Should be 4 space indentation, did you not get any linting reporting in Sublime Text? Might be error'ing on this file like it has for me? Still needs work.

Also could you add this to the Shell Grid Test case: http://campaignmonitor.github.io/shell/test/grid.html?